internal/services/mobileapi/mobileapi.go

```diff
@@ -0,0 +1,72 @@
+package mobileapi
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type Client struct {
+	base *url.URL
+	http *http.Client
+}
+
+var zeroAddr common.Address
+
+func New(base *url.URL) *Client {
+	return &Client{
+		base: base,
+		http: &http.Client{},
```

Comment:
Maybe this needs a timeout, or the context needs a timeout, I don't know. I'm not too worried about it.